openedx/frontend-app-admin-portal
# feat: add group selection to invite members modal

# Description
Updates the members invite modal to include a dropdown menu with flex groups. 
https://2u-internal.atlassian.net/browse/ENT-9490

# UI Changes
![Screenshot 2024-11-01 at 2 02 05 PM](https://github.com/user-attachments/assets/eeff8112-10a7-4e35-8d04-f66205e609b9)

# Testing
1. checkout branch `knguyen2/ent-9490` and visit link https://localhost.stage.edx.org:1991/exec-ed-2u-integration-qa/admin/learner-credit/e259a8e5-e0ad-4c9d-bf14-9abf310158c7
2. confirm that you see a dropdown menu
3. select a group and confirm that the email populates in the summary section

# For all changes


# Only if submitting a visual change
